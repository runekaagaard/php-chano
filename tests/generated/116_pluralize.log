
---- EXPECTED OUTPUT
s
---- ACTUAL OUTPUT
Fatal error: Call to undefined method DtlIter::pluralize() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/116_pluralize.php on line 8
---- FAILED
