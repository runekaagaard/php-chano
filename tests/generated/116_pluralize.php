<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => array (
  0 => 1,
  1 => 2,
  2 => 3,
))));
foreach ($items as $i) echo $i->input->pluralize();
