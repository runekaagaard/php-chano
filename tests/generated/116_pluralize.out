Fatal error: Call to undefined method DtlIter::pluralize() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/116_pluralize.php on line 8